packages/lab/src/__tests__/__e2e__/overlay/Overlay.cy.tsx

```diff
@@ -112,4 +118,26 @@ describe("GIVEN an Overlay", () => {
       });
     });
   });
+
+  describe("WHEN controlled", () => {
+    it("THEN it should call onOpenChange on enter when trigger has focus", () => {
+      const onOpenChangeSpy = cy.stub().as("onOpenChangeSpy");
+      cy.mount(<Default open={false} onOpenChange={onOpenChangeSpy} />);
+
+      cy.realPress("Tab");
+      cy.realPress("Enter");
+      cy.get("@onOpenChangeSpy").should("have.callCount", 1);
+    });
+
+    it("THEN it should call the onOpenChange on CloseButton click and outside overlay click", () => {
+      const onOpenChangeSpy = cy.stub().as("onOpenChangeSpy");
+      cy.mount(<Default onOpenChange={onOpenChangeSpy} open />);
```

Comment:
is `open` actually a controlled prop? if so, are we actually testing flipping `open` would show/hide the component?

Reply:
update the controlled story - thanks :)